portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -201,8 +201,6 @@ definition {
 				value1 = "ES Element");
 
 			Portlet._clickAddPortlet(portletName = "ES Element");
-
-			Refresh();
 		}
```

Comment:
I have removed a refresh from here as well, as a result of consulting it with the dev: the change should be applied immediately after adding the widget to the page.

Reply:
I think we had CI stability issues at this step in the past. Not sure if it got better but ok to try and see.